packaging/makeself/jobs/20-openssl.install.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# shellcheck source=packaging/makeself/functions.sh
+. "$(dirname "${0}")/../functions.sh" "${@}" || exit 1
+
+version="$(cat "$(dirname "${0}")/../openssl.version")"
+
+export LDFLAGS='-static'
+export PKG_CONFIG="pkg-config --static"
+
+run git clone git://git.openssl.org/openssl.git "${NETDATA_MAKESELF_PATH}/tmp/openssl"
```

Comment:
Do this in one step and save some bandwith with `--single-branch`?
`git clone --branch <tag_name> <repo_url> --signle-branch`